app/models/order_import.rb

```diff
@@ -196,6 +198,7 @@ def set_error_mode
   def store_error_report
     self.error_file = StoredFile.new(
       file: StringIO.new(self.error_report),
+      file_content_type: "text/csv",
```

Comment:
Looks like existing files are stored with the wrong mime-type: 

```
StoredFile.where(name: 'error_report.csv').group(:file_content_type).count
   (4.0ms)  SELECT COUNT(*) AS count_all, file_content_type AS file_content_type FROM "STORED_FILES" WHERE "STORED_FILES"."NAME" = 'error_report.csv' GROUP BY file_content_type
 => {"text/plain"=>268, "text/x-c++"=>14, "text/x-pascal"=>4, "text/x-java"=>6}
```

We should probably have a rake task or migration that cleans these up so they get downloaded as the correct thing.